### PR TITLE
added fairseq seq-2-seq models

### PR DESCRIPTION
**Pull request by swapaj**
* Added Truecaser model to convert wrong casing text to correct casing (for English)
* Added character stream to text generation model. Converts space-separated characters to readable, correct text.

### CODE REVIEW

File: nlp/seq2seq/util.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+###
```
Comment (venkata-ramireddy-mettu) [inline nlp/seq2seq/util.py:1]: Why are we adding this model in **util.py** file?

Comment (swapaj) [reply nlp/seq2seq/util.py:1]: `Seq2SeqFairseqWrapper` is a kind of utility. Any seq-2-seq model can be used through this. In future ... we can use something else than fairseq and just add a wrapper class as a utility in it.

**Review by sanprit — APPROVED** [review]

File: nlp/seq2seq/util.py (hunk 1 above)
Comment (venkata-ramireddy-mettu) [reply nlp/seq2seq/util.py:1]: Ok.

**Review by venkata-ramireddy-mettu — APPROVED** [review]
Looks good.

**Pull request merged**